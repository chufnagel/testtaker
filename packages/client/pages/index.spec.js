/* global beforeEach describe, expect, test */
import { shallow } from "enzyme";
import Index from "./index";

describe("Index", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Index />);
  });
  test("It should render without crashing", () => {
    expect(wrapper.exists()).toEqual(true);
  });
  test("Index matches snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
