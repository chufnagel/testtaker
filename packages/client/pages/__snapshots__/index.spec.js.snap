// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Index matches snapshot 1`] = `
<Layout>
  <Passage
    passage={
      Array [
        Object {
          "line": "1",
          "text": "Ergo iter inceptum peragunt fluvioque propinquant.",
        },
        Object {
          "line": "2",
          "text": "Navita quos iam inde ut Stygia prospexit ab unda",
        },
        Object {
          "line": "3",
          "text": "per tacitum nemus ire pedemque advertere ripae,",
        },
        Object {
          "line": "4",
          "text": "Line sic prior adgreditur dictis atque increpat ultro:",
        },
        Object {
          "line": "5",
          "text": "“Quisquis es, armatus qui nostra ad flumina tendis,",
        },
        Object {
          "line": "6",
          "text": "fare age, quid venias, iam istinc et comprime gressum.",
        },
        Object {
          "line": "7",
          "text": "Umbrarum hic locus est, somni noctisque soporae:",
        },
        Object {
          "line": "8",
          "text": "corpora viva nefas Stygia vectare carina.",
        },
        Object {
          "line": "9",
          "text": "Nec vero Alciden me sum laetatus euntem",
        },
        Object {
          "line": "10",
          "text": "accepisse lacu, nec Thesea Pirithoumque,",
        },
        Object {
          "line": "11",
          "text": "dis quamquam geniti atque invicti viribus essent.”",
        },
      ]
    }
  />
</Layout>
`;
